src/ImageSharp/PixelFormats/Generated/PixelOperations{TPixel}.Generated.cs

```diff
@@ -10,320 +10,379 @@ namespace SixLabors.ImageSharp.PixelFormats
-        /// A helper for <see cref="PackFromRgba64(ReadOnlySpan{Rgba64}, Span{TPixel}, int)"/> that expects a byte span.
-        /// The layout of the data in 'sourceBytes' must be compatible with <see cref="Rgba64"/> layout.
+
+        /// <summary>
+        /// A helper for <see cref="PackFromArgb32(ReadOnlySpan{Argb32}, Span{TPixel}, int)"/> that expects a byte span.
+        /// The layout of the data in 'sourceBytes' must be compatible with <see cref="Argb32"/> layout.
         /// </summary>
         /// <param name="sourceBytes">The <see cref="ReadOnlySpan{T}"/> to the source bytes.</param>
         /// <param name="destPixels">The <see cref="Span{T}"/> to the destination pixels.</param>
         /// <param name="count">The number of pixels to convert.</param>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        internal void PackFromRgba64Bytes(ReadOnlySpan<byte> sourceBytes, Span<TPixel> destPixels, int count)
+        internal void PackFromArgb32Bytes(ReadOnlySpan<byte> sourceBytes, Span<TPixel> destPixels, int count)
         {
-            this.PackFromRgba64(MemoryMarshal.Cast<byte, Rgba64>(sourceBytes), destPixels, count);
+            this.PackFromArgb32(MemoryMarshal.Cast<byte, Argb32>(sourceBytes), destPixels, count);
         }
-		
-		/// <summary>
-        /// Converts 'count' pixels in 'sourcePixels` span to a span of <see cref="Rgba64"/>-s.
-        /// Bulk version of <see cref="IPixel.ToRgba64(ref Rgba64)"/>.
+
+        /// <summary>
+        /// Converts 'count' pixels in 'sourcePixels` span to a span of <see cref="Argb32"/>-s.
         /// </summary>
         /// <param name="sourcePixels">The span of source pixels</param>
-        /// <param name="dest">The destination span of <see cref="Rgba64"/> data.</param>
+        /// <param name="destPixels">The destination span of <see cref="Argb32"/> data.</param>
         /// <param name="count">The number of pixels to convert.</param>
-        internal virtual void ToRgba64(ReadOnlySpan<TPixel> sourcePixels, Span<Rgba64> dest, int count)
+        internal virtual void ToArgb32(ReadOnlySpan<TPixel> sourcePixels, Span<Argb32> destPixels, int count)
         {
-            GuardSpans(sourcePixels, nameof(sourcePixels), dest, nameof(dest), count);
+            GuardSpans(sourcePixels, nameof(sourcePixels), destPixels, nameof(destPixels), count);
 
             ref TPixel sourceBaseRef = ref MemoryMarshal.GetReference(sourcePixels);
-            ref Rgba64 destBaseRef = ref MemoryMarshal.GetReference(dest);
+            ref Argb32 destBaseRef = ref MemoryMarshal.GetReference(destPixels);
 
             for (int i = 0; i < count; i++)
             {
                 ref TPixel sp = ref Unsafe.Add(ref sourceBaseRef, i);
-                ref Rgba64 dp = ref Unsafe.Add(ref destBaseRef, i);
-                sp.ToRgba64(ref dp);
+                ref Argb32 dp = ref Unsafe.Add(ref destBaseRef, i);
+
+                dp.PackFromScaledVector4(sp.ToScaledVector4());
```

Comment:
This is again a significant performance regression. Many pixel formats had efficient implementations for the conversion of basic formats like (`Rgba32` <-> `Bgra32` <-> `Argb32` <-> `Rgb24` etc, implemented with byte-shuffling.

We should not replace those with expensive internal `Vector4` conversions!